src/app/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+
+Sitemap: https://www.oghunt.com/sitemap.xml
```

Comment:
www what the fuck is that, probably should be the non redirected domain which is www less

PLS FIX 🙏

Reply:
shut up and merge
![image](https://github.com/user-attachments/assets/2e121ce4-0b58-40c8-ae38-8db0d1682c03)

Reply:
updated